docs/e2etests.md

```diff
@@ -169,7 +169,33 @@ Format:
     "id": "receiver-1"
   }
 }
+```
```

Comment:
@michaljurecko Mohl by ses vlastně podívat předběžně aspoň na tenhle commit (https://github.com/keboola/keboola-as-code/pull/1193/commits/bd9cd6bbf8a2f6a7c1773dbc8f7ec3eeff1f1ac5) kde je přidaný to pollování do testů a upravenej jeden test, jestli k tomu nemáš případně už nějaký připomínky.

Reply:
Toto vyzera velmi pekne, ... ked tam mas vyriesene aj nahradenie toho `https://buffer.keboola.local` -> `http://localhost:port` tak super.

Reply:
(pozrel som zatial iba tu docs)

Reply:
Mno já si to zjednodušil. Rozparsoval jsem url a vzal z ní jen path 🙂 Může být?

Reply:
Jasne, mozno by to bolo dobre spomenut v tom readme.

Reply:
jo 👍